ua-converter/ua_convert.py

```diff
@@ -64,35 +114,96 @@ def process_item(item):
         "device_brand": ua_result.device.brand if ua_result.device else None,
         "browser": ua_result.user_agent.family if ua_result.user_agent else None,
         "browser_version": browser_version,
-        "browser_version_major_minor": browser_version_major_minor,
+        "version": browser_version_major_minor,
         "os": ua_result.os.family if ua_result.os else None,
         "os_version": os_version,
-        "platform": item["platform"],
+        "system": item["platform"],
```

Comment:
It kept `platform` matched to platform on-purpose, since I actually think we should rename `system` in our code base, since that is just a wrong name.